book/controller.rst

```diff
@@ -778,11 +784,25 @@ There are also special classes to make certain kinds of responses easier:
   :class:`Symfony\\Component\\HttpFoundation\\StreamedResponse`.
   See :ref:`streaming-response`.
 
-.. seealso::
+JSON helper
+~~~~~~~~~~~
 
-    Now that you know the basics you can continue your research on Symfony
-    ``Request`` and ``Response`` object in the
-    :ref:`HttpFoundation component documentation <component-http-foundation-request>`.
+Returning JSON contents is increasingly popular for API-based applications. For
+that reason, the base controller class defines a ``json()`` method which creates
+a ``JsonResponse()`` and encodes the given contents automatically::
+
+    // ...
+    public function indexAction()
+    {
+        // returns '{"username":"jane.doe"}' and sets the proper Content-Type header
+        $data = array('username' => 'jane.doe');
+        return $this->json($data);
+    }
+
+The only required argument is the data to be sent, but ``json()`` defines three
```

Comment:
I've removed the paragraph ... but updated the code because otherwise `json()` looks "useless" (people may think that you can't change the status code, the headers, etc.)